app/models/revision.rb

```diff
@@ -61,9 +64,19 @@ def init
     # Helper for recording a notebook revision
     def notebook_commit(revtype, notebook, user, message)
       return nil unless GalleryConfig.storage.track_revisions
-      commit_id = GitRepo.add_and_commit(notebook, message)
+      if GalleryConfig.storage.database_notebooks
+        commit_id = Digest::SHA1.hexdigest(notebook.content + user.user_name + message + notebook.uuid + DateTime.current.to_s)
+        Rails.logger.debug("Revision Content" + notebook.notebook.to_git_format(notebook.uuid))
+        notebookFile = NotebookFile.new(save_type: "revision", content: notebook.notebook.to_git_format(notebook.uuid), uuid: notebook.uuid)
```

Comment:
It doesn't matter to me either way, but we may not *need* to do the git_format conversion here anymore since it's not actually going into git.  But if it simplifies other code elsewhere, then I have no objection.

Reply:
You know what, I thought there were a bunch of places that did the to/from git content but it turns out it is only the revision.content functions, so I will take a look at this.